src/score.js

```diff
@@ -0,0 +1,47 @@
+class Score {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+    dataScore = [];
+
+    baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:id/scores/';
+
+getScore = () => {
+  const listScore = document.querySelector('.list');
+  listScore.innerHTML = this.dataScore.map((el, index) => `<div class=${index % 2 !== 0 ? 'list1' : 'list2'}>${el.user}:${el.score}</div>`).join('');
+}
+
+fetchScore = async () => {
+  try {
+    const data = await fetch(this.baseUrl);
+    const response = await data.json();
+    response.result.map((item) => this.dataScore.push(item));
```

Comment:
- [ ] Please make sure to reset the dataScore to an empty array as when you push inside it you are adding the data again inside it or you can optimize it without using the map function you can use the spread operator example: `dataScore = [...response.result];`.